### ISSUE TITLE
Hi can we get a accuracy of Every Named Entity in Spacy

### ISSUE THREAD

**Issue by chowdary-git** [explosion/spaCy#3891]
## Could the feature be a [custom component](https://spacy.io/usage/processing-pipelines#custom-components) or [spaCy plugin](https://spacy.io/universe)?
If so, we will tag it as [`project idea`](https://github.com/explosion/spaCy/labels/project%20idea) so other users can take it on.

**Comment by chowdary-git** [comment]
Hi when we perform the custom entity recognition,we can see the overall training loss .Can we also see how accurately every entity is recognized in train and validation sets

**Comment by ines** [comment]
Merging this with #2882.
